app/views/coronavirus_local_restrictions/results.html.erb

```diff
@@ -17,12 +17,18 @@
     %>
     <% if !@search.location_lookup.data.first.england? %>
       <%= render partial: "coronavirus_local_restrictions/devolved_nation", locals: locals %>
-    <% elsif @search.local_restriction&.tier_two? %>
+    <% end %>
+
+    <% if @search.local_restriction&.tier_one? %>
```

Comment:
I think these conditions should be kept as if/elsif so we reduce the need of always having to evaluate all of them.

Reply:
ah yeah! I have changed back.